doc/src/Developer_updating.rst

```diff
@@ -0,0 +1,304 @@
+Notes for updating code written for older LAMMPS versions
+---------------------------------------------------------
+
+This section documents how C++ source files that were written for an
```

Comment:
Clarify that this section is for users who have their own private LAMMPS extensions which they wish to keep synced with current LAMMPS.  I.e. not for code previously contributed to public LAMMPS.

Reply:
done